src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
```

Comment:
You should use variables. (checklist)
I guess, it will be nice to create variables for 20, 40, 50